cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The 'include' function is using the 'namespace' argument, but it requires the corresponding app's 'urls.py' to define an 'app_name' variable. Ensure that the 'cinema.urls' and 'user.urls' modules define 'app_name' to match the namespaces used here.